### PR TITLE
updated display name to <brand> v2 in all python packages v2 integrat…

### PR DESCRIPTION
**Pull request by orhovy**
…ions

## Status
- [x] Ready

## Description
Changed in v2 integrations the display name to <brand> v2

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Integrations/KennaV2/KennaV2.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ configuration:
 - defaultvalue: 'false'
   display: Trust any certificate (not secure)
   name: insecure
-  required: false
+  required: False
```
Comment (bakatzir) [inline Integrations/KennaV2/KennaV2.yml]: rollback

**Comment by bakatzir** [comment]
@amshamah419 Can you help us with the RN? WDYT?

**Comment by bakatzir** [comment]
related to: https://github.com/demisto/etc/issues/21244

**Comment by Itay4** [comment]
why do we need rn for these?

**Comment by bakatzir** [comment]
On the one hand, very small change. on the other - very visible.
@kirbles19 WDYT?

**Pull request closed without merging**